Sources/BraintreeThreeDSecure/BTPaymentFlowClient+ThreeDSecure.m

```diff
@@ -127,12 +127,12 @@ - (void)performThreeDSecureLookup:(BTThreeDSecureRequest *)request
                     [self.apiClient sendAnalyticsEvent:@"ios.three-d-secure.lookup.network-connection.failure"];
                 }
                 // Provide more context for card validation error when status code 422
-                if ([error.domain isEqualToString:BTHTTPError.domain] &&
-                    error.code == BTHTTPErrorCodeClientError &&
-                    ((NSHTTPURLResponse *)error.userInfo[BTHTTPError.urlResponseKey]).statusCode == 422) {
+                if ([error.domain isEqualToString:BTCoreConstants.httpErrorDomain] &&
+                    error.code == 2 &&
```

Comment:
This doesn't feel _great_ but is what we will need to do until this module is in Swift. We cannot have access to the Swift enum in this module because the enum contains associated values that cannot be represented (or exposed) to Obj-C. Alternatively we could extract the client error into an`Int` property on `BTCoreConstants` and use that both here and in the enums `errorCode` for this value (similar to what we do for the domain). Thoughts?

Reply:
Maybe we can add a comment for readability that the 2 here matches with `BTHTTPError.errorCode.clientError`?